

<launch>
    <arg name="config"             default="configs/condlane/resnet101_culane.py" />
    <arg name="load_from"          default="condlane_r101_culane.pth" />
    <arg name="input_image_topic"  default="camera1" />
    <arg name="output_image_topic" default="image_lane_detection" />
    <arg name="result_topic"       default="detected_lanes" />

    <node name="lane_detector" pkg="lane_detector" type="detect_node.py">
        <param name="config"             type="str" value="$(arg config)" />
        <param name="load_from"          type="str" value="$(arg load_from)" />
        <param name="input_image_topic"  type="str" value="$(arg input_image_topic)" />
        <param name="output_image_topic" type="str" value="$(arg output_image_topic)" />
        <param name="result_topic"       type="str" value="$(arg result_topic)" />
    </node>
</launch>
